client/src/components/AdvancedSearchbar.jsx

```diff
@@ -0,0 +1,35 @@
+import { InputGroup, FormControl, Form } from "react-bootstrap";
+
+export const AdvancedSearchbar = ({ changeHandler }) => {
+  let states = [ "State", "AK","AL","AR","AS","AZ","CA","CO","CT","DC","DE","FL","GA","GU","HI","IA","ID","IL","IN","KS","KY","LA","MA","MD","ME","MI","MN","MO","MS","MT","NC","ND","NE","NH","NJ","NM","NV","NY","OH","OK","OR","PA","PR","RI","SC","SD","TN","TX","UT","VA","VI","VT","WA","WI","WV","WY"];

```

Comment:
We could make this a `const` (and eventually move it into a file storing all constant values in the client)